backend/src/docs/v1.api-spec.yaml

```diff
@@ -41,32 +41,72 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/appConfig/{webAdeEnv}/{acronym}':
+  '/webAde/{webAdeEnv}/{acronym}/appConfig/':
     get:
-      summary: Returns WebADE JSON response for the specified acronym in the specified environment
+      summary: >-
+        Returns WebADE JSON response for the specified acronym in the specified
+        environment
```

Comment:
You have the ability to add a `description` property at the same level as summary if you want to provide a more detailed explanation. It doesn't all necessarily need to be in the summary.

Reply:
Noted. This one is short enough I think.